spec/dummy/config/initializers/flex_commerce_api.rb

```diff
@@ -0,0 +1,5 @@
+FlexCommerceApi.config do |config|
+  config.flex_root_url ||= 'http://shiftcommmerce-api:3000'
+  config.flex_account = 'testaccount'
+  config.flex_api_key = '314e47ece578a11e6e77c6153229cd25'
```

Comment:
It's fake, yes – it prevents `flex-ruby-gem` from throwing errors. I may change it to just be "test" or something similar so it's obvious that it's mock data.